src/components/common/styles/spinner.js

```diff
@@ -0,0 +1,9 @@
+import { StyleSheet } from 'react-native';
```

Comment:
Define this inside the component's folder

Reply:
See #6